src/uwtools/tests/config/test_validator.py

```diff
@@ -121,6 +124,14 @@ def test_validate_yaml_pass(config, config_file, schema, schema_file):
     assert validator.validate_yaml(schema_file=schema_file, config_file=config_file)
 
 
+def test_validate_workflow_tags_pass():
+    # Test a fully valid config file passes schema
+    with resources.as_file(resources.files("uwtools.resources")) as resc:
+        schema_file = resc / "rocoto.jsonschema"
+    config_file = support.fixture_path("hello_workflow_tags.yaml")
+    assert validator.validate_yaml(schema_file=schema_file, config_file=config_file)
+
+
```

Comment:
I utilized the `validate_yaml` method here that already "checks whether the given config file (yaml) conforms to the given JSON Schema spec and whether any filesystem paths it identifies do not exist." I found this to be a good place to add the unit test since it seemed to fit in nicely. Let me know if you'd like to pull it out.

One thing to note is once we have each subtask from UW-320 working properly and merged into the `rocoto.jsonschema` then we can remove these individual fixtures such as `hello_workflow_tags` and we can soimply test against `hello_workflow`. Then we can change our unit test above to reflect this as well.

Reply:
> we can remove these individual fixtures ... and we can simply test against ...
Yes! As we start to get more realistic schemas and test data, we should be able to remove some placeholder tests data and concentrate on the realistic stuff. Nice.